scripts/generate-key

```diff
@@ -8,16 +8,21 @@
 
 # The signing key is generally used in "ed25519.signingkey" in the sydent config
 
-import sys
-
 import signedjson.key
 
-signing_key = signedjson.key.generate_signing_key(0);
+signing_key = signedjson.key.generate_signing_key(0)
 sk_str = "%s %s %s" % (
     signing_key.alg,
     signing_key.version,
     signedjson.key.encode_signing_key_base64(signing_key)
     )
-print ("signing key: %s " % sk_str)
 pk_str = signedjson.key.encode_verify_key_base64(signing_key.verify_key)
-print ("verify key: %s" % pk_str)
+
+print(
+    "# A new key has been generated. To use it, update your sydent config file with the following: \n"
```

Comment:
This is really neither one thing nor the other. It has a `#`, presumably to make it safe to include in the config file; but it would be very confusing to find this text as a comment in the config file.

I suggest just making people copy it manually for now. If you're keen to have an automated option, you can always make it take a `-q` ("quiet") commandline option or something.

Reply:
I've added a --quiet option